src/redist/redist.csproj

```diff
@@ -200,41 +198,72 @@
           Condition=" '$(DISABLE_CROSSGEN)' == '' "
           AfterTargets="PublishMSBuildExtensions">
     <ItemGroup>
+      <RoslynFiles Include="$(PublishDir)Roslyn\bincore\**\*" />
```

Comment:
This seems like it will place all of our compiler binaires inside bincore. How do our MSBuild task binaries get published?